cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -36,14 +36,15 @@ spec:
       properties:
         spec:
           properties:
+            annotations: {}
             cephVersion:
               properties:
                 allowUnsupported:
                   type: boolean
                 image:
                   type: string
                 name:
-                  pattern: ^(luminous|mimic|nautilus)$
+                  pattern: ^(luminous|mimic|nautilus|octopus)$
```

Comment:
not so sure about this one!

Reply:
Don't worry about this, i'm removing validation of the `name` in #3450 already. That property was removed a while back.